### DISCUSSION TITLE
[ FF7 ] using ffnx with 7th heaven causes gamepad configurations to stop working

### DISCUSSION THREAD

**Discussion by sisco1986** [julianxhokaxhiu/FFNx#171] [Q&A]
i did a clean 7th heaven + ffnx install without mods and when i activate ffnx the gamepad config from 7th heaven stops working and i get a strange controller configuration where X is B, B is A and A is X (using a x360 gamepad) i tried to use every single control profile and every single one is overwritten with that strange profile with swapped buttons 

this happens with a untouched FFNx.cfg and using 7th heaven v2.0.3.406 + FFNx-FF7_1998-v1.7.2.0, if i change the dll to the normal 7H_GameDriver.dll controls start working again instantly

**Comment by julianxhokaxhiu** [comment]
Hi,

Any 7h 2.x `ff7input.cfg` file is simply wrong. Make sure you start the game without any file configured, and you'll get the correct layout.

Feel free to approach the 7h 2.x author in order to fix that.

Cheers,
Julian

**Comment by mangopirate** [comment]
I also experienced this issue only after installing FFNx with 7h 2.x. 

I solved it by going to 7h Game Launcher Settings and setting the controls to 1998 Original. Seems like the default setting which is 1998 KB+Std Gamepad was causing the issue.

**Comment by julianxhokaxhiu** [comment]
Good to know. Thanks for the tip!
